index.html

```diff
@@ -427,7 +427,7 @@ <h3>
       independent browser engines, to meet the W3C's adequate implementation
       experience requirements.
       </li>
-      <li>MUST be defined in a specification which is available freely and publicly
+      <li>MUST be defined in a specification which is available publicly
```

Comment:
There was not consensus on the call about this.

Reply:
I think it's fair to say there's not full consensus either way. But there has been support expressed by a number of folks in various places in this PR for inclusion of "freely".

Reply:
By my count, there have been numerous individuals from a diverse set of organizations expressing support for the requirement that an entry in the registry be freely available and two individuals from a single organization saying that listing pay-walled specifications is okay.

Despite that count, the e11c2f70fb39f07092146b2554883dda79397df6 change removes the word "freely".